docs/src/udp.rst

```diff
@@ -312,6 +312,29 @@ API
 
     .. versionchanged:: 1.27.0 added support for connected sockets
 
+.. c:function:: int uv_udp_send_ex(uv_udp_send_t* req, uv_udp_t* handle, const uv_buf_t bufs[], unsigned int nbufs, const struct sockaddr* addr, unsigned int addrlen, uv_udp_send_cb send_cb)
+
+    Same as :c:func:`uv_udp_send`, but accepts any `struct sockaddr`.
```

Comment:
Maybe suggest why / how this is different, since `uv_udp_send` also takes a `struct sockaddr` so it might be all that clear at the first glance.